File: Spec/Level01/ex-5-bootreceiver/src/main/java/ru/skillbranch/ex_5_bootreceiver/MainActivity.java
package ru.skillbranch.ex_5_bootreceiver;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.d("happy", "1111");
    }

    public void sendSomething(View view) {
        sendBroadcast(new Intent("SMS"));
    }
}

File: WFMS/MySelfStudy/app/src/main/java/ru/vetukov/wfms/myselfstudy/MainActivity.kt
package ru.vetukov.wfms.myselfstudy

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*
import ru.vetukov.wfms.myselfstudy.Utils.Utils

class MainActivity : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        main_btn_enter.setOnClickListener(this)
    }

    override fun onClick(v: View?) {
        val intent = Intent(this, BaseActivity::class.java)
        intent.putExtra(Utils.EMAIL_FIELD, main_et_email.text)
        intent.putExtra(Utils.PWD_FIELD, main_et_pwd.text)
        startActivity(intent)
    }
}
